e3sm_to_cmip/mpas.py

```diff
@@ -260,7 +267,7 @@ def write_netcdf(ds, fileName, fillValues=netCDF4.default_fillvals):
     variableNames = list(ds.data_vars.keys()) + list(ds.coords.keys())
     for variableName in variableNames:
         isNumeric = numpy.issubdtype(ds[variableName].dtype, numpy.number)
-        if isNumeric and numpy.any(numpy.isnan(ds[variableName])):
+        if isNumeric:
```

Comment:
Even if there aren't any fill values, we add the appropriate `_FillValue` attribute.  This is important for remapping.